README.md

```diff
@@ -3,54 +3,72 @@
 
 # OpenQA Public cloud Helper
 
-A web app which monitors, displays and deletes CSPs and linked resources. It tries to identify
-left over instances and also performs some cleanup task. For instance it deletes old custom
-uploaded images. Behavior differs per CSP.
+Publiccloud-Watcher or pcw is a a web app which monitors, displays and deletes resources on various cloud service providers (CSPs) and linked resources. It identifies abandoned instances by searching for certain tags and performs cleanup tasks. It also deletes old custom uploaded images. Behavior differs per CSP.
```

Comment:
```suggestion
PublicCloud-Watcher or pcw is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs). It identifies abandoned instances by searching for certain tags and performs cleanup tasks. It also deletes old custom uploaded images. The behavior differs per CSP.
```
not sure what you mean about `... and linked resources`

Reply:
Yeah, that's too vague, especially for the first paragraph. I removed it as you suggested, that reads nice.